packages/account/src/Sections/Security/SelfExclusion/self-exclusion.jsx

```diff
@@ -482,9 +483,12 @@ class SelfExclusion extends React.Component {
                                                         <Localize
                                                             i18n_default_text='You’ll be able to adjust these limits at any time. You can reduce your limits from the <0>self-exclusion page</0>. To increase or remove your limits, please contact our <1>Customer Support team</1>.'
                                                             components={[
-                                                                <span
+                                                                <Text
                                                                     key={0}
-                                                                    className='self-exclusion__text-highlight'
+                                                                    size='xs'
+                                                                    color='loss-danger'
+                                                                    weight='bold'
+                                                                    text_align='left'
```

Comment:
Isn't this redundant?

Reply:
No, I removed these styles from css, this task is for handling simple text styles using Text component